zap_log_test.go

```diff
@@ -221,7 +221,7 @@ func TestWithOptions(t *testing.T) {
 func TestLogJSON(t *testing.T) {
 	ts := newTestLogSpy(t)
 	conf := &Config{Level: "debug", DisableTimestamp: true, Format: "json"}
-	logger, _, _ := InitTestLogger(ts, conf, zap.AddStacktrace(zapcore.FatalLevel))
+	logger, _, _ := InitTestLogger(ts, conf, zap.AddStacktrace(zapcore.FatalLevel), zap.AddCallerSkip(-1))
```

Comment:
Yep, this change will break the caller of the package...

Reply:
yes , I will use `AddCallerSkip(1)` in first calling .